packages/api/db/migration/20230124222902_update_support_type_enum.js

```diff
@@ -0,0 +1,15 @@
+export const up = async function (knex) {
+  await knex.schema.alterTable('supportTicket', () => {
+    knex.raw(`ALTER TYPE support_type ADD VALUE 'Delete my farm'`);
+  });
+};
+
+export const down = async function (knex) {
+  await knex.schema.alterTable('supportTicket', () => {
+    knex.raw(`
+            CREATE TYPE support_type1 AS ENUM ('Request information', 'Report a bug', 'Request a feature', 'Other');
```

Comment:
I read that you cannot remove a value from an enum and found this approach. Not tested